cmd/main.go

```diff
@@ -3,14 +3,16 @@ package main
 import (
 	"flag"
 	"io"
-	"log"
 	"os"
 	"os/signal"
 	"syscall"
 	"time"
 
+	"github.com/lvl484/positioning-filter/logger"
+
 	"github.com/lvl484/positioning-filter/config"
 	"github.com/lvl484/positioning-filter/storage"
+	log "github.com/sirupsen/logrus"
```

Comment:
Do you have any conflict with to same "log"

Reply:
fixed